Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -25,7 +25,8 @@ class BrowserViewController: UIViewController {
     private var webViewContainer: UIView!
     private let uriFixup = URIFixup()
     private var screenshotHelper: ScreenshotHelper!
-    private var homePanelIsInline: Bool = false
+    private var homePanelIsInline = false
+    private var searchLoader: SearchLoader!
```

Comment:
Its only used while the SearchViewController is open though right? Why keep it around all the time?